test/utils/RandomDataGenerator.test.js

```diff
@@ -0,0 +1,105 @@
+/* eslint no-plusplus: ["error", { "allowForLoopAfterthoughts": true }] */
+const faker = require('faker')
+const RandomDataGenerator = require('../../src/utils/RandomDataGenerator')
+const RegexUtils = require('../../src/utils/RegexUtils')
+
+describe('the RandomDataGenerator', () => {
+  const numberOfTestRuns = 100
+
+  describe('.uuid', () => {
+    it('should return a uuid', () => {
+      for (let index = 0; index < numberOfTestRuns; index++) {
+        const uuid = RandomDataGenerator.uuid()
+        expect(typeof (uuid)).toBe('string')
+        expect(uuid).toMatch(RegexUtils.UUIDV4)
+      }
```

Comment:
if you want to move away from this more procedural code, you can do this: 

```js
const uuids = Array.from({ length: numberOfTestRuns }, () => RandomDataGenerator.uuid())
uuids.forEach(uuid => {
  expect(typeof (uuid)).toBe('string')
  expect(uuid).toMatch(RegexUtils.UUIDV4)
})
```

Then you also will not need your eslint-rule at the top of file